assets/dev/scss/frontend/widgets/tabs.scss

```diff
@@ -114,7 +114,6 @@
 
 	.elementor-tab-title {
 		cursor: pointer;
-		outline: none;
```

Comment:
For now, keep this rule

Reply:
Done in this commit:
https://github.com/elementor/elementor/pull/13264/commits/88858537213c18fd571339361f3e8a6fd20a6514